dev/setup-fleet

```diff
@@ -40,6 +40,9 @@ helm -n cattle-fleet-system upgrade --install --create-namespace --wait --reset-
   --set apiServerCA="$ca" \
   --set apiServerURL="$server" \
   $shards_settings \
+  --set controller.reconciler.workers.gitrepo=50 \
+  --set controller.reconciler.workers.bundle=50 \
+  --set controller.reconciler.workers.bundledeployment=50 \
```

Comment:
If we need it here (do we, if those are the new default values?), then we'd need it in `.github/scripts/deploy-fleet.sh` as well so that it applies to our testing workflows.

Reply:
We don't need, but I wanted to expose if for testing. Should I remove that?

Reply:
Oh I see, thanks for clarifying. No need then.